src/pages/VKeywordCounter.vue

```diff
@@ -0,0 +1,248 @@
+<script setup lang="ts">
+import { ref, computed, onMounted, watch } from 'vue';
+import { MaCard, MaInput, MaButton } from '@mobileaction/action-kit';
+
+interface Props {
+  text?: string;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  text: ''
+});
+
+const inputText = ref('');
+
+const totalWords = computed(() => {
+  return textToWords(inputText.value).length;
+});
+
+const wordCount = computed(() => {
+  const words = textToWords(inputText.value);
+  return words.reduce((acc, word) => {
+    acc[word] = (acc[word] || 0) + 1;
+    return acc;
+  }, {} as { [key: string]: number });
+});
+
+const keywordStats = computed(() => {
+  if (totalWords.value === 0) return [];
+  
+  return Object.entries(wordCount.value)
+    .map(([keyword, count]) => ({
+      keyword,
+      count,
+      percentage: Math.round((count / totalWords.value) * 100 * 100) / 100
+    }))
+    .sort((a, b) => b.count - a.count);
+});
+
+const textToWords = (text: string) => {
+  if (!text.trim()) return [];
+  
+  return text
+    .toLowerCase()
+    .replace(/[^\w\s]/g, ' ')
+    .replace(/\s+/g, ' ')
+    .trim()
+    .split(' ')
+    .filter(word => word.length > 0);
+};
+
+const initializeText = () => {
+  if (props.text) {
+    inputText.value = props.text;
+  }
+};
+
+onMounted(initializeText);
```

Comment:
```suggestion
onMounted(() => {
	initializeText()
});
```

When you have multiple operations you can chain them that way